server/subscribe.go

```diff
@@ -22,10 +22,16 @@ const (
 	JIRA_SUBSCRIPTIONS_KEY = "jirasub"
 )
 
+type SubscriptionFilters struct {
+	Event     []string `json:"event"`
```

Comment:
naming: 1/5 `s/Event/Events/`?

Reply:
The filters are currently all singular:
Project: list of project ids
IssueType: list of issue types
Event: list of event types

I'm thinking they should all agree with each other. Do you think they should all be plural since they are lists? This is something we need to do before we release `2.1`.

Reply:
I usually pluralize lists.